vllm/utils.py

```diff
@@ -20,12 +20,13 @@
 import numpy as np
 import psutil
 import torch
+import torch.types
```

Comment:
can you check if this file is type checked from format.sh? seems like it didn't before, if so can you add it?